tools/restatectl/src/commands/replicated_loglet/info.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2024 -  Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use anyhow::Context;
+use cling::prelude::*;
+
+use restate_admin::cluster_controller::protobuf::cluster_ctrl_svc_client::ClusterCtrlSvcClient;
+use restate_admin::cluster_controller::protobuf::ListLogsRequest;
+use restate_cli_util::{c_indentln, c_println};
+use restate_types::logs::metadata::Logs;
+use restate_types::replicated_loglet::ReplicatedLogletId;
+use restate_types::storage::StorageCodec;
+use restate_types::Versioned;
+use tonic::codec::CompressionEncoding;
+
+use crate::app::ConnectionInfo;
+use crate::util::grpc_connect;
+
+#[derive(Run, Parser, Collect, Clone, Debug)]
+#[cling(run = "get_info")]
+pub struct InfoOpts {
```

Comment:
No strong opinion her but we've used the verb "describe" for this type of command elsewhere - we can solve with aliases and the internal struct doesn't need to be consistent, but it may be nice for users to see the same set of sub-commands everywhere.